src/commands/phone-numbers/update.js

```diff
@@ -82,7 +81,18 @@ class NumberUpdate extends TwilioClientCommand {
       try {
         newBaseUrl = await this.ngrok.connect(newTunnel);
       } catch (error) {
-        throw new TwilioCliError((error.details && error.details.err) || error);
+        this.logger.debug('ngrok response: ' + JSON.stringify(error));
+        if (isHttps && error.details.err.includes('too many colons in address')) {
+          // We don't have a current ngrok version downloaded
```

Comment:
`ngrok` is _installed_ under `node_modules/ngrok/bin/ngrok` when `twilio-cli` is installed. When npm installing:
```
> npm i -g twilio-cli
...
> ngrok@3.2.5 postinstall ... /lib/node_modules/twilio-cli/node_modules/ngrok
> node ./postinstall.js

ngrok - downloading binary https://bin.equinox.io/c/4VmDzA7iaHb/ngrok-stable-darwin-amd64.zip
ngrok - downloading progress: 13671591/13671591
ngrok - binary downloaded to ~/.ngrok/aHR0cHM6Ly9iaW4uZXF1aW5veC5pby9jLzRWbUR6QTdpYUhiL25ncm9rLXN0YWJsZS1kYXJ3aW4tYW1kNjQuemlw.zip
ngrok - unpacking binary
ngrok - binary unpacked to ... /lib/node_modules/twilio-cli/node_modules/ngrok/bin/ngrok
```
For brew, it's included in that CLI tar so it's whatever version was downloaded when the CLI was packaged-up.

Reply:
Yeah, I tried calling the postInstall script directly, but it's turned into a rabbit hole.